src/main.c

```diff
@@ -226,6 +234,12 @@ static void configure(Data *d) {
         sizeof(headlights_off_konami_sequence)
     );
 
+    if (d->float_conf.tiltback_bms_enabled) {
+        d->bms_fault = BMSF_CONNECTION;
+    } else {
+        d->bms_fault = BMSF_NONE;
+    }
```

Comment:
Why this? IMO it should be initialized to `NONE` and only set the connection error on a timeout.